packages/snaps-sdk/src/jsx/components/Link.ts

```diff
@@ -6,7 +6,7 @@ import type { StandardFormattingElement } from './formatting';
  * The children of the {@link Link} component.
  */
 export type LinkChildren = MaybeArray<
```

Comment:
Same question with regards to types, would it be easier to have a shared type that adds `boolean | null` and whatever else